docs/pages/management/guides/awsoidc-integration.mdx

```diff
@@ -0,0 +1,163 @@
+---
+title: AWS OIDC Integration
+description: How to connect your AWS account with Teleport and provide access to AWS resources.
+---
+
+This guide explains how to set up the Teleport AWS OIDC Integration.
+It targets users who would prefer a more manual approach or which to use a more Infrastrcutre as Code oriented flow.
+
+As an alternative to this guide, you can use the Teleport Web UI (Access Management / Enroll New Integration).
+
+With an integration you will no longer need to deploy Teleport Agents in AWS manually for most use cases.
+The following features use an Integration to interact with AWS:
+- [External Audit Storage](../../choose-an-edition/teleport-cloud/external-audit-storage.mdx)
+- RDS Auto Discovery
+- EC2 Auto Discovery
+- [Access Graph AWS Sync](../../access-controls/access-graph/aws-sync.mdx)
+
+## How it works
+Teleport is added as an Identity Provider in AWS by exposing the `openid-configuration` and public keys in a public S3 bucket.
+
+This allows Teleport to assume a configured Role and access AWS resources.
```

Comment:
Any service or even agent can access the AWS resources.

Auth, Proxy and Discovery service can access any service.

While other agents/clients can only call our own APIs (which end up calling the AWS APIs), as long as they have the `integration.use` allow rule
https://github.com/gravitational/teleport/blob/0a26cdc5ef528244b2369cf6fc9a98bbf97dff6b/api/proto/teleport/integration/v1/awsoidc_service.proto#L24

Do you think it's worth adding this explanation?
IMO, it's implementation details and makes this guide a bit complex without providing any actual product/feature value, so I don't think we should add it

Reply:
As an operator, I'm not sure the "How it works" section would help me understand the workloads that this feature adds to a Teleport cluster: how do different Teleport components communicate in order to make this work? Does the OIDC authorization server run as part of the Teleport Auth Service, for example, or is there a separate Teleport service you need to deploy?

Reply:
Communication is over gRPC as usual and there's no extra Teleport Service that you need to deploy/configure/maintain.

I can be explicit about this feature not requiring a specific service.
IMO, for simplicity purposes I don't think we should add that, given that it is not a requirement.
Let me know what you think.

Reply:
Yeah, I would make it explicit that the Teleport Auth Service runs the OIDC authorization server (assuming that's true—I'm guessing from the source). Otherwise, for a cluster with thousands of `teleport` processes running as agents, Auth Service replicas or Proxy Service replicas, "Teleport" obscures what's going on in an actual deployment.

Reply:
There's no operational overhead, this service works just like the Roles or Users Service, and is irrelevant from the operator standpoint.

The token generation happens in Auth Service, but I don't think it's worth adding that here.

Having said that, what do you think about:
> This runs alongside your existing cluster and no extra configuration or services are required.